Geometry/CMSCommonData/python/cmsExtendedGeometry2017XML_cfi.py

```diff
@@ -207,21 +206,18 @@
         'Geometry/EcalCommonData/data/eecon.xml',
         'Geometry/EcalCommonData/data/eefixed.xml',
         'Geometry/EcalCommonData/data/eehier.xml',
-        'Geometry/EcalCommonData/data/ectkcable.xml',
         'Geometry/EcalCommonData/data/eealgo.xml',
         'Geometry/EcalCommonData/data/escon.xml',
         'Geometry/EcalCommonData/data/esalgo.xml',
         'Geometry/EcalCommonData/data/eeF.xml',
         'Geometry/EcalCommonData/data/eeB.xml',
         'Geometry/HcalCommonData/data/hcalrotations.xml',
-        'Geometry/HcalCommonData/data/PhaseI/hcalalgo.xml',
+        'Geometry/HcalCommonData/data/hcalalgo.xml',
```

Comment:
@boudoul - I see you are rolling back alignment with SLHC release:
https://github.com/makortel/cmssw/commit/767313c55f476f37c3539ea889de5cf610242b00#diff-9c452f0e342fb87194f2f19ceac82e3f